csrc/fused_moe/cutlass_backend/cutlass_fused_moe_kernels.cuh

```diff
@@ -259,7 +259,8 @@ void buildMinLatencyActiveExpertMaps(int* num_active_experts_per_node,
                                      int const experts_per_token, int const start_expert,
                                      int const end_expert, int const num_experts_per_node,
                                      int const cluster_rank, int const cluster_size,
-                                     int const num_experts_smem, cudaStream_t const stream) {
+                                     int const num_experts_smem, cudaStream_t const stream,
+                                     bool enable_pdl = false) {
```

Comment:
`stream` should be the last argument, and please do not set a default value for `enable_pdl`